piker/ui/_exec.py

```diff
@@ -200,6 +200,7 @@ async def main():
                 tractor._root._default_arbiter_port,
             ),
             name='qtractor',
+            start_method='trio',
```

Comment:
I wish we had more peeps to test this 😂 

Would love to force this always.

Reply:
@wattygetlood actually, can you try without this after the history loading config changes i suggested?

Reply:
Wtv we like `trio` more anyway 😂